Loop/Managers/DeviceDataManager.swift

```diff
@@ -66,7 +66,7 @@ final class DeviceDataManager {
 
     private(set) var servicesManager: ServicesManager!
 
-    var analyticsServicesManager: AnalyticsServicesManager { return servicesManager.analyticsServicesManager }
+    var analyticsServicesManager: AnalyticsServicesManager
 
     var loggingServicesManager: LoggingServicesManager { return servicesManager.loggingServicesManager }
```

Comment:
Curious why `analyticsServicesManager` gets special treatment, but `loggingServicesManager` and `remoteDataServicesManager` still reach into another manager?

Reply:
Those are still tied up in a dependency cycle of DDM -> LDM -> stores -> RDM.  There is a bunch more refactoring that needs to happen here, but I'm holding off, as there are too many in-flight changes between DIY (significant changes in LDM) and Tidepool (significant changes in DDM) atm.  I am targeting the next refactoring for after the next sync point.

Reply:
Actually, I think `loggingServicesManager` should be created and owned by `DeviceDataManager` just like `analyticsServicesManager`.

Reply:
Oh, yeah, LogginServicesManager is not dependent on the stores either. Changing this. Thanks.